tests/ui-fulldeps/session-diagnostic/enforce_slug_naming.rs

```diff
@@ -19,6 +19,6 @@ use rustc_errors::{Applicability, MultiSpan};
 extern crate rustc_session;
 
 #[derive(Diagnostic)]
-#[diag(compiletest_example, code = "E0123")]
+#[diag(compiletest_example, code = 0123)]
```

Comment:
Shouldn't this be `E0123`?

Reply:
Good question! I investigated, turns out it doesn't matter. This test (deliberately) fails to compile because the slug name has the incorrect form. Apparently that error occurs earlier than the type error that we would get for the code.

I can fix it in a follow-up, but it's not urgent.

Reply:
I figured it did not matter much, but since I noticed thought it's better to comment :)